src/db/tripledb.plt

```diff
@@ -0,0 +1,23 @@
+:- begin_tests('db/tripledb').
+
+:- use_module(library('db/tripledb'),
+    [ tripledb_load/2,
+      tripledb_ask/3
+    ]).
+
+test('tripledb_load(URL)') :-
+  tripledb:tripledb_load('https://protege.stanford.edu/ontologies/pizza/pizza.owl',
+    [ graph(common),
+      namespace(pizza,'https://protege.stanford.edu/ontologies/pizza/#')
```

Comment:
there are no entities in this namespace!
use instead:

    namespace(pizza,'http://www.co-ode.org/ontologies/pizza/pizza.owl#')

and remove rdf_register_ns call below.

Reply:
If I use simple namespace(pizza,'http://www.co-ode.org/ontologies/pizza/pizza.owl#'), it does not work!

Reply:
you replaced below "code_pizza:" with "pizza:", right?